src/eia930.py

```diff
@@ -402,6 +403,7 @@ def manual_930_adjust(raw: pd.DataFrame):
             - PJM-OVEC, all time. Based on OVEC demand - generation, OVEC
                 should be a net exporter to PJM
                 - Note: OVEC's data repeats daily starting in 2018...
+                - Note: I think OVEC data is actually being added by `gridemissions` -- there's no EIA-930 data starting in 2018
```

Comment:
It looks like there are two places where OVEC is mentioned in the `gridemissions` code:
1. In `src.gridemissions.eia_columns.csv`
2. In `src.gridemissions.eia_api.py` in the `BAs` list at the top. 

However, at first glance, it appears that these mentions are only related to scraping EIA-930 data, which I don't think we are doing, so not sure why it is showing up....

Reply:
Seems like this is a non-issue; OVEC actually *isn't* showing up in our outputs/930/930_x results. I think it's just in the manual notebook and code because the first time I did this I was using the `gridemissions` scraping functions, which inserted OVEC. But it's not relevant now that we're using the balance files. Removed.